sdk/core/core-rest-pipeline/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
     "outDir": "./dist-esm",
-    "declarationDir": "./types/latest"
+    "declarationDir": "./types/latest",
+    "lib": ["dom"]
   },
-  "exclude": ["node_modules", "types", "temp", "browser", "dist", "dist-esm", "./samples/**/*.ts"]
+  "include": ["src/**/*.ts", "test/**/*.ts"]
```

Comment:
Switched to `include` instead of `exclude` to prevent `core-rest-pipeline.shims.d.ts` from being detected as an ambient component of the TypeScript build.